server/athenian/api/models/db/github/__init__.py

```diff
@@ -0,0 +1,223 @@
+from sqlalchemy import ARRAY, BigInteger, Boolean, Column, DateTime, Integer, String, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class VersionedMixin:
+    versions = Column("versions", ARRAY(Integer()), index=True)
+
+
+class IssueComment(Base, VersionedMixin):
+    __tablename__ = "github_issue_comments_versioned"
+    sum256 = Column("sum256", String(64), primary_key=True)
```

Comment:
Noticed the `sum256` is used frequently. Maybe deserves a `Mixin` as well?

Reply:
+1

Reply:
https://github.com/athenianco/athenian-api/issues/6